namecheap.py

```diff
@@ -108,7 +108,11 @@ def _fetch_xml(self, payload, extra_payload = None):
             print(r.text)
         xml = fromstring(r.text)
 
-        if xml.attrib['Status'] == 'ERROR':
+        # Ensure that API response is in valid format, see: https://github.com/Bemmu/PyNamecheap/issues/8
+        if 'Status' not in xml.attrib:
```

Comment:
Maybe write this as:
`if xml.get('Status') is None:`

(the ElementTree doc says that `.get()` is preferable to `.attrib[]`)